test/success/J.prl

```diff
@@ -0,0 +1,28 @@
+Def J/square(#i:dim,#j:dim, #ty, #a, #p) = [
+  (hcom 0~>#j #ty #a
+   [#i=0 [_] #a]
+   [#i=1 [j] (@ #p j)])
+].
+
+Print Welp.
```

Comment:
It's strange to me that MLton on travis is okay with this... `Welp` is not defined.

Reply:
@favonia Hmm, that's interesting... I would like to track that down.